app/templates/content/index.hbs

```diff
@@ -97,12 +97,18 @@
                         </div>
                     {{/if}}
 
-                    {{#if model.doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        {{#if model.doi}}
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
                             <a href={{doiUrl}} onlick={{action "click" "link" "Content - DOI" doiUrl}}>{{model.doi}}</a>
-                        </div>
-                    {{/if}}
+                        {{else if (not node.public)}}
+                            {{t 'content.preprint_pending_doi'}}
+                        {{else if (and model.provider.reviewsWorkflow (not model.isPublished))}}
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
+                            {{t 'content.preprint_pending_doi_moderation'}}
+                        {{/if}}
```

Comment:
This logic belongs in the section above this one that deals with `model.links.preprint_doi`